common/script_values/wc_men_at_arms_values.txt

```diff
@@ -81,6 +83,22 @@ culture_ai_weight_caster = {
 		}
 		add = -60
 	}
+	if = { # Cultures with a special MaA of that type should avoid the base version of the unit
+		limit = {
+			culture = {
+				OR = {
+					has_innovation = innovation_archmage
+					has_innovation = innovation_chaosborn
+					has_innovation = innovation_hexer
+					has_innovation = innovation_magi
+					has_innovation = innovation_naga_army
+					has_innovation = innovation_subjagators
+					has_innovation = innovation_technomage
+			}
```

Comment:
This file misses a lot of brackets. It's reported in `error.log`.

Reply:
👌